nirc_ehr/resources/referenceStudy/study/datasets/datasets_manifest.xml

```diff
@@ -10,6 +10,7 @@
         <dataset name="breeder" id="1008" category="Colony Management" type="Standard"/>
         <dataset name="clinremarks" id="1009" category="Colony Management" type="Standard"/>
         <dataset name="cases" id="1023" category="Clinical" type="Standard"/>
+        <dataset name="casesTemp" id="1026" category="Clinical" type="Standard"/>
```

Comment:
I don't like this being a study dataset. I imagine it will show up in places like Browse All Datasets, where we don't want it listed. I would be more comfortable with this being a table in the nirc_ehr schema, but that may be a bigger refactor than we want to tackle now.

There's a `showByDefault` attribute on `<dataset>` that should hide it from some places.

Reply:
Changed this to a table